context/app/static/js/components/workspaces/SelectableTemplateGrid/SelectableTemplateGrid.tsx

```diff
@@ -43,6 +44,26 @@ function SelectableTemplateGrid<FormType extends FormWithTemplates>({
     field.value satisfies FormType[typeof inputName],
   );
 
+  // Sort templates alphabetically by title with selected templates first, then default template
+  const sortedTemplates = useMemo(
```

Comment:
Given the added complexity, want to pull the sort function out of the component and add a test?

Reply:
For sure! Would this go in `js/helpers/functions`?

Reply:
I think it can go in a `utils.ts` file in the same directory as the component. The test can go in `utils.spec.ts`.